.github/workflows/go-check.yml

```
@@ -27,7 +27,7 @@ jobs:
             echo "RUNGOGENERATE=true" >> $GITHUB_ENV
           fi
       - name: Install staticcheck
-        run: go install honnef.co/go/tools/cmd/staticcheck@c8caa92bad8c27ae734c6725b8a04932d54a147b # 2021.1.2 (v0.2.2)
+        run: go install honnef.co/go/tools/cmd/staticcheck@d7e217c1ff411395475b2971c0824e1e7cc1af98 # 2022.1 (v0.3.0)
```

Nit: If there is no specific reason to use the commit hash maybe is more clear to use the version number instead.

```suggestion
        run: go install honnef.co/go/tools/cmd/staticcheck@v0.3.1
```